spec/models/dhs1171_pdf_spec.rb

```diff
@@ -45,8 +55,8 @@ def filled_in_values(file:)
     end
   end
 
-  def generated_pdf
-    @_generated_pdf ||= "tmp/dhs1171_test_output.pdf"
+  def new_pdf_filename
+    @_new_pdf_filename ||= "dhs1171_test_output.pdf"
```

Comment:
ah I see. So I think my confusion here underscores the confusion of passing in a variable but not having it reflect the full file path. Maybe rather than calling the argument `filename` we should call it `file_path` and include `tmp/`?